denoise.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+import numpy as np
+import wave
+import math
+
+def nextpow2(n):
+    return np.ceil(np.log2(np.abs(n))).astype('long')
+
+
+
+f = wave.open("denoise/output.wav")
+# (nchannels, sampwidth, framerate, nframes, comptype, compname)
+params = f.getparams()

```

Comment:
I think the `with` context should be used here too for variable `f`

Reply:
Thanks for help! I will fix it as soon as possible!